cpp/ps0003/ps0003_solv.cpp

```diff
@@ -4,23 +4,23 @@ using namespace std;
 
 int main()
 {
-    int x,counter;
+    // prompting user for number 2/2
+    int x, counter;
     cout << "Please enter the number of attempts \n";
-    cin>>counter;
-    for(int i=1;i<=counter;i++)
+    cin >> counter;
+    
+    for(int i = 1; i <= counter; i++)
     {
       cout << "\n Please enter the number\t";
-      cin>>x;
-      if(x!=0)
-      {
-         if(x%2==0)
+      cin >> x;

```

Comment:
- [x] declared a variable to store users input
- [x] used `std::cin` to prompt user for the number